src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
It is better not to specify the type for the shape and color as a string.

Because you know what expect in shape and color.

create
``type Shape = 'triangle' | 'circle' | 'rectangle';``
and
``type Color = 'red' | 'green' | 'blue';``

Now you can reusing it in other places.